mhkit/wave/io/hindcast.py

```diff
@@ -0,0 +1,109 @@
+import pandas as pd
+import numpy as np
+from rex import WaveX, MultiYearWaveX
+
+def request_wpto_dataset(data_type, parameter, lat_lon, years, tree=None, 
+                                 unscale=True, str_decode=True,hsds=True):
+    
+        """
+        Accesses data from the WPTO wave hindcast data hosted on AWS. 
```

Comment:
Currently the dataset covers the west coast of the US, but it is going to be expanded in iterations over the next couple years. I am planning on adding a link to the hindcast data website when @aidanbharath and his team have finished it. That way users can look there to see what locations are available, rather than us having to update the docString every time new data gets added.

Reply:
I think that if you are planning to update the docstring to a website later then you could remove the reference to west coast at the same time and it would be helpful to know until that time. If you don't want to add it here I think we should say the west coast economic exclusion zone in the example notebook.